admyral/db/schemas/control_schemas.py

```diff
@@ -0,0 +1,103 @@
+from typing import TYPE_CHECKING
+from sqlmodel import (
+    Field,
+    Relationship,
+    TEXT,
+    UniqueConstraint,
+    ForeignKeyConstraint,
+)
+
+from admyral.db.schemas.base_schemas import BaseSchema
+from admyral.models import Control, ControlsWorkflowsMapping
+
+if TYPE_CHECKING:
+    from admyral.db.schemas.auth_schemas import UserSchema
+    from admyral.db.schemas.workflow_schemas import WorkflowSchema
+
+
+class ControlsWorkflowsMappingSchema(BaseSchema, table=True):
+    """
+    Schema for Controls Workflows Mapping
+    """
+
+    __tablename__ = "controls_workflows"
+    __table_args__ = (
+        ForeignKeyConstraint(
+            ["control_id"],
+            ["controls.control_id"],
+            ondelete="CASCADE",
+        ),
+        ForeignKeyConstraint(
+            ["workflow_id"],
+            ["workflows.workflow_id"],
+            ondelete="CASCADE",
+        ),
+    )
+
+    # primary keys
+    control_id: int = Field(primary_key=True)
+    workflow_id: str = Field(sa_type=TEXT(), primary_key=True)
+
+    # relationships
+    control: "ControlSchema" = Relationship(back_populates="control_workflow_mappings")
+    workflow: "WorkflowSchema" = Relationship(
+        back_populates="control_workflow_mappings"
+    )
+
+    def to_model(self) -> ControlsWorkflowsMapping:
+        return ControlsWorkflowsMapping.model_validate(
+            {
+                "control_id": self.control_id,
+                "workflow_id": self.workflow_id,
+            }
+        )
+
+
+class ControlSchema(BaseSchema, table=True):
+    """
+    Schema for Control
+    """
+
+    __tablename__ = "controls"
+    __table_args__ = (
+        UniqueConstraint("user_id", "control_id", name="unique_control_id"),
+        ForeignKeyConstraint(
+            ["user_id"],
+            ["User.id"],
+            ondelete="CASCADE",
+        ),
+    )
+
+    # primary keys
+    control_id: int = Field(
+        primary_key=True,
+        index=True,
+    )
+
+    # foreign keys
+    user_id: str = Field(sa_type=TEXT(), index=True)
```

Comment:
Lets make `user_id` part of the primary key, so we can easily swap it with an `org_id`/`workspace_id` later